editor/editor_inspector.cpp

```diff
@@ -4087,7 +4090,32 @@ void EditorInspector::_notification(int p_what) {
 			changing++;
 
 			if (update_tree_pending) {
-				update_tree();
+				// if the new property list is the same as the old one, we don't need
+				// to re-create everything.
+				bool same_property_list = true;
+				List<PropertyInfo> newlist;
+				object->get_property_list(&newlist, true);
+				if (last_object == object) {
+					List<PropertyInfo>::Element *OLD_property = property_list.front();
+					for (List<PropertyInfo>::Element *NEW_property = newlist.front(); NEW_property; NEW_property = NEW_property->next()) {
+						if (!OLD_property) {
+							same_property_list = false;
+							break;
+						}
+						PropertyInfo &old_p = OLD_property->get();
+						PropertyInfo &new_p = NEW_property->get();
+						if (!(old_p == new_p)) {
```

Comment:
```suggestion
						if (old_p != new_p) {
```

Reply:
Thanks, I would have done this but this raises the compilation error, `no match for 'operator!='`, I thought maybe `C++`  may infer this as the inverse of the == operator on `PropertyInfo` but this doesn't appear to be the case. Would you like me to add a != operator on `PropertyInfo` as part of this PR?